content/en/community/hackathons/sessions/baby-steps/content/Demos_01.-Building-and-Running-the-Helloworld-Application.md

```diff
@@ -7,12 +7,12 @@ The `kraft up` command makes it easy to do that with one swoop.
 Let's do that for the `helloworld` application (listed with `kraft list`):
 ```
 $ kraft up -t helloworld hello
- 100.00% :::::::::::::::::::::::::::::::::::::::: |       21 /       21 |:  app/helloworld@0.5
-[INFO    ] Initialized new unikraft application: /home/razvan/hello
-make: Entering directory '/home/razvan/.unikraft/unikraft'
+ 100.00% :::::::::::::::::::::::::::::::::::::::: |       21 /       21 |:  app/helloworld@0.10.0
```

Comment:
I think we should rename these files to by demo_XY_short_description

Reply:
The current naming convention seems to apply to all other sessions as well. I think for the moment we should keep it like that for consistency.